docs/src/main/asciidoc/central-logging.adoc

```diff
@@ -97,18 +101,19 @@ services:
 networks:
   graylog:
     driver: bridge
-```
+----
 
 Then, you need to create a UDP input in Graylog.
 You can do it from the Graylog web console (System -> Input -> Select GELF UDP) or via the API.
 
 This curl example will create a new Input of type GELF UDP, it uses the default login from Graylog (admin/admin).
 
-```
+[source,shell script]
```

Comment:
I think it should be just `shell`.

Reply:
This is what IDEA proposes.
I try to generate the doc with both and this does not seems to be any difference so I replace by script.